controllers/discovery/initDiscoveryUserData.js

```diff
@@ -122,6 +124,7 @@ const InitDiscoveryUserData = async (req, res) => {
       type: QueryTypes.SELECT,
       replacements: {
         userId,
+        // topicIds: topicIds || [],
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The commented out line `// topicIds: topicIds || [],` can be removed to keep the code clean and avoid confusion for future developers.
```diff
- // topicIds: topicIds || [],
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->